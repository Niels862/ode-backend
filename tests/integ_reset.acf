chip {
    io: [
        input,
        input,
    ],
    cabs: [
        cab 1 with clocks 1, 3 {
            cams: [
                Integrator as integ1 {
                    reset: true,
                    integ_const: 8,
                },
            ],
        },
        cab 2 with clocks 1, 3 {
            cams: [
                Integrator as integ2 {
                    reset: true,
                    integ_const: 8,
                },
            ],
        },
        cab 3 with clocks 1, 3 {
            cams: [
                Integrator as integ3 {
                    reset: true,
                    integ_const: 8,
                },
            ],
        },
        cab 4 with clocks 1, 3 {
            cams: [
                Integrator as integ4 {
                    reset: true,
                    integ_const: 8,
                },
            ],
        },
    ],
    routing: [
        io1 -> integ1,
        io2 -> integ1:cmp,
        
        io1 -> integ2,
        io2 -> integ2:cmp,
        
        io1 -> integ3,
        io2 -> integ3:cmp,
        
        io1 -> integ4,
        io2 -> integ4:cmp,
    ],
}
