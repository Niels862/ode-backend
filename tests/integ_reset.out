0 0 0 0 32 4 0 2 21 0 0 64 0 0 81 255 15 241 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 255 0 1 0 16 0 1 0 16 0 0 64 0 0 64 0 0 64 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 254 0 0 0 0 0 0 8 0 0 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 236 0 0 0 0 0 64 0 0 7 201 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 108 5 2 16 2 24 0 0 0 0 0 0 127 254 0 0 0 0 0 0 8 0 0 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 236 0 0 0 0 0 64 0 0 7 201 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 108 5 2 16 2 24 0 0 0 0 0 0 127 254 0 0 0 0 0 0 8 0 0 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 236 0 0 0 0 0 4 0 0 7 201 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 108 5 2 16 2 24 0 0 0 0 0 0 127 254 0 0 0 0 0 0 8 0 0 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 236 0 0 0 0 0 1 0 0 7 201 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 108 5 2 16 2 24